specification/cognitiveservices/data-plane/CustomSearch/v1.0/CustomSearch.json

```diff
@@ -0,0 +1,607 @@
+					"$ref": "#/definitions/QueryContext",
+					"readOnly": true
+				},
+				"webPages": {
+					"description": "A list of webpages that are relevant to the search query.",
+					"$ref": "#/definitions/WebWebAnswer",
+					"readOnly": true
+				}
+			}
+		},
+		"Response": {
+			"description": "Defines a response. All schemas that could be returned at the root of a response should inherit from this",
+			"allOf": [{
+				"$ref": "#/definitions/Identifiable"
+			}],
+			"type": "object",
+			"properties": {
+				"webSearchUrl": {
+					"description": "The URL To Bing's search result for this item.",
+					"readOnly": true,
+					"type": "string"
+				}
+			}
+		},
+		"QueryContext": {
+			"description": "Defines the query context that Bing used for the request.",
+			"type": "object",
+			"required": [
+				"originalQuery"
+			],
+			"properties": {
+				"originalQuery": {
+					"description": "The query string as specified in the request.",
+					"type": "string"
+				},
+				"alteredQuery": {
+					"description": "The query string used by Bing to perform the query. Bing uses the altered query string if the original query string contained spelling mistakes. For example, if the query string is \"saling downwind\", the altered query string will be \"sailing downwind\". This field is included only if the original query string contains a spelling mistake.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"alterationOverrideQuery": {
+					"description": "The query string to use to force Bing to use the original string. For example, if the query string is \"saling downwind\", the override query string will be \"+saling downwind\". Remember to encode the query string which results in \"%2Bsaling+downwind\". This field is included only if the original query string contains a spelling mistake.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"adultIntent": {
+					"description": "A Boolean value that indicates whether the specified query has adult intent. The value is true if the query has adult intent; otherwise, false.",
+					"readOnly": true,
+					"type": "boolean"
+				},
+				"askUserForLocation": {
+					"description": "A Boolean value that indicates whether Bing requires the user's location to provide accurate results. If you specified the user's location by using the X-MSEdge-ClientIP and X-Search-Location headers, you can ignore this field. For location aware queries, such as \"today's weather\" or \"restaurants near me\" that need the user's location to provide accurate results, this field is set to true. For location aware queries that include the location (for example, \"Seattle weather\"), this field is set to false. This field is also set to false for queries that are not location aware, such as \"best sellers\".",
+					"readOnly": true,
+					"type": "boolean"
+				},
+				"isTransactional": {
```

Comment:
>isTransactional [](start = 5, length = 15)

Why not include a description?

Reply:
isTransactional was actually an unused param. I've removed it.